nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-administration/src/main/java/org/apache/nifi/authorization/StandardAuthorizerConfigurationContext.java

```diff
@@ -44,8 +47,13 @@ public String getIdentifier() {
     }
 
     @Override
-    public String getProperty(String property) {
-        return properties.get(property);
+    public PropertyValue getProperty(String property) {
+        final String value = properties.get(property);
+        if (value == null) {
```

Comment:
In this case, I think we should still be returning a new StandardPropertyValue with a value of null, rather than returning no PropertyValue at all. This allows us to chain method calls, such as: Integer x = getProperty(...).asInteger();